.gitlab-ci.yml

```diff
@@ -0,0 +1,90 @@
+image: golang:1.11
+
+cache:
+  paths:
+    - /apt-cache
+    - /go/src/github.com
+    - /go/src/golang.org
+    - /go/src/gopkg.in
+    - node_modules
+
+variables:
+  DOCKER_HOST: tcp://docker:2375/
+  DOCKER_DRIVER: overlay2
+
+services:
+  - docker:dind
+  - mongo
+
+stages:
+  - test
+
+before_script:
+  - export DOCKER_REPOSITORY="mendersoftware/workflows"
+  - export DOCKER_TAG=${CI_COMMIT_REF_SLUG:-master}
+  - export SERVICE_IMAGE=$DOCKER_REPOSITORY:$DOCKER_TAG
+  - export COMMIT_TAG="$CI_COMMIT_REF_SLUG"_"$CI_COMMIT_SHA"
+
+test:static:
+  stage: test
+  dependencies: []
+  services: []
+  script:
+    # Install cyclomatic dependency analysis tool
+    - go get -u github.com/fzipp/gocyclo
+    - curl -sL https://deb.nodesource.com/setup_11.x | bash -
+    - apt-get -qq update
+    - apt-get install -qy --allow-unauthenticated python3-pip nodejs liblzma-dev
+    - pip3 install pyyaml
+    - npm install -g swagger-cli
+    # Get our own Swagger verifier
+    - wget https://raw.githubusercontent.com/mendersoftware/autodocs/master/verify_docs.py
```

Comment:
This repo does not have API docs. This is unnecessary

Reply:
Yes it does :)